### ISSUE TITLE
Item Mode Switch log spam error

### ISSUE THREAD

**Issue by SamsyTheUnicorn** [mekanism/Mekanism#6549]
#### Issue description:
When "Item Mode Switch" in controls doesn't have a key bound to it begins logging the following line repeatedly: "[Render thread/ERROR] [net.minecraft.client.MainWindow/]: 65539: Invalid key -1", this results in the logs being bloated.

#### Steps to reproduce:
1. Use a launcher like MultiMC to track the logs live.
2. Remove the keybind from "Item Mode Switch" in Controls.
3. Observe error messages.

#### Versions:
**Forge:** Forge v32.0.108
**Mekanism:** Mekanism 1.16.1-10.0.9.432

**Comment by pupnewfster** [comment]
Does this happen with other mods keys if you unbind them or even vanilla keys? Given I am pretty sure we are just using the normal keybinding system, unless maybe there is a place we are checking it and mojang changed the behavior of unbound keys erroring when checking if pressed instead of returning they aren't pressed.

**Comment by SamsyTheUnicorn** [comment]
I unbind all the keybinds added by mods at the start of working on a modpack so at first, I wasn't sure what mod was the cause.
For troubleshooting, I set all mod-added controls to a single key, semi-colon for my test, and then one by one unbind them using the escape key, went back into the game, and checked the logs, by doing this I confirmed that the only keybind causing this error to appear was the "Item Mode Switch" keybind.

To test even further I created an instance with the following mods enabled and Forge v32.0.108:
- Controlling 7.0.0.3
- Mekanism 10.0.9.432
- MekanismAddition ^^^
- MekanismGenerators ^^^
- MekanismTools ^^^
I launched this new instance and made a new world with the default options, then I unbind all the Mekanism keybinds except "Item Mode Switch", check logs, no errors. I then unbind "Item Mode Switch" and the following occurs every tick (or second):

> [m[1;31m[03:21:24] [Render thread/ERROR] [minecraft/MainWindow]: ########## GL ERROR ##########
[m[1;31m[03:21:24] [Render thread/ERROR] [minecraft/MainWindow]: @ Pre render
[m[1;31m[03:21:24] [Render thread/ERROR] [minecraft/MainWindow]: 65539: Invalid key -1

Here is a link to a Paste.ee upload of the log: [https://paste.ee/p/diAuj#s=0&l=907](url)

Let me know if there's anything further I can do to help.
